go/cmd/vtctld/vtctld.go

```diff
@@ -269,6 +257,11 @@ func main() {
 	})
 
 	// vschema editor
+	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Yes, I would image the vtctld page will be a single angular app. There will be always a navigation bar on the top, each tab renders page in a different way but they all live under host:port/#...